src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -347,6 +354,58 @@ describe('run-test', () => {
           testTimeout: 42,
         },
         publicIds: ['ran-dom-id'],
+        // TODO SYNTH-12989: Clean up `pollingTimeout` in favor of `batchTimeout`
+        pollingTimeout: 1, // also set for backwards compatibility
+        subdomain: 'new-sub-domain',
+        testSearchQuery: 'a-search-query',
+        tunnel: true,
+      })
+
+      // TODO SYNTH-12989: Merge those 2 test cases when `pollingTimeout` is removed
+      command['batchTimeout'] = 2 // when both are used, `batchTimeout` takes precedence
+      await command['resolveConfig']()
+      expect(command['config']).toEqual({
+        ...DEFAULT_COMMAND_CONFIG,
+        apiKey: 'fake_api_key',
+        appKey: 'fake_app_key',
+        batchTimeout: 2,
+        configPath: 'src/commands/synthetics/__tests__/config-fixtures/empty-config-file.json',
+        datadogSite: 'datadoghq.eu',
+        failOnCriticalErrors: true,
+        failOnMissingTests: true,
+        failOnTimeout: false,
+        files: ['new-file'],
+        jUnitReport: 'junit-report.xml',
+        defaultTestOverrides: {
+          allowInsecureCertificates: true,
+          basicAuth: {
+            password: 'password',
+            username: 'username',
+          },
+          body: 'a body',
+          bodyType: 'bodyType',
+          cookies: {
+            value: 'name1=value1;name2=value2;',
+            append: true,
+          },
+          defaultStepTimeout: 42,
+          deviceIds: ['chrome.laptop_large'],
+          executionRule: ExecutionRule.BLOCKING,
+          followRedirects: true,
+          headers: {'Content-Type': 'application/json', Authorization: 'Bearer token'},
+          locations: ['us-east-1'],
+          mobileApplicationVersion: '00000000-0000-0000-0000-000000000000',
+          mobileApplicationVersionFilePath: './path/to/application.apk',
+          retry: {
+            count: 5,
+            interval: 42,
+          },
+          startUrl: 'startUrl',
+          startUrlSubstitutionRegex: 'startUrlSubstitutionRegex',
+          testTimeout: 42,
```

Comment:
You'll also need a `pollingTimeout` here 👍 ~This would also show that the nested `pollingTimeout` does not get updated like the one at the root~ (I saw you show this in the next test already)